README.md

```diff
@@ -100,7 +100,7 @@ We use the div to control the entire height and width, and the span to get acces
 * `stateful`: true/false. If true, then whenever a column width is resized, it will store it in html5 localStorage, if available. Defaults to false.
 * `contextmenu`: true/false whether or not a context menu for editing the cells should be shown on right-click. Defaults to false.
 * `gridcontextmenu`: function to create context menu items; see context menu of jstree for format. In addition, if `false` or not set, no special context menu. If `true`, creates a default menu to edit the cell entry.
-
+* `caseInsensitive`: true/false whether or not the column sort should be case sensitive or insensitive
```

Comment:
There is a formatting error here. Need an extra newline between the `caseInsensitive` bullet point and the new paragraph. 

Also, please indicate the default.